src/app/containers/FrontPageSection/utilities/storyRowsSplitter.jsx

```diff
@@ -0,0 +1,36 @@
+import { splitEvery, splitAt } from 'ramda';
```

Comment:
These are being imported with destructing in some places like this and `ramda/src/path` in others. Is tree-shaking taking care of the above example when bundled? I'd prefer the above syntax.

Reply:
By above I mean `import { splitEvery, splitAt } from 'ramda';` :)

Reply:
It doesn't seem to be taking care of it. We've updated the imports to use `ramda/src/module` and the bundle got reduced.

Before:
![image](https://user-images.githubusercontent.com/46446236/74666955-5f113c80-519a-11ea-9e7a-67b9ddf5c0fd.png)

After:
<img width="472" alt="image2" src="https://user-images.githubusercontent.com/46446236/74666974-67697780-519a-11ea-96dc-94baf9e5c84c.png">